comm/src/main/scala/coop/rchain/catscontrib/ski/package.scala

```diff
@@ -0,0 +1,10 @@
+package coop.rchain.catscontrib
+
+package object ski {
+  def σ[A, B, C](x: A => B => C, y: A => B, z: A): C         = x(z)(y(z))
+  def κ[A, B](x: => B): A => B                               = _ => x
+  def κ2[A, B, C](x: => C): (A, B) => C                      = (_, _) => x
+  def κ3[A, B, C, D](x: => D): (A, B, C) => D                = (_, _, _) => x
+  def κ6[A, B, C, D, E, F, G](x: G): (A, B, C, D, E, F) => G = (_, _, _, _, _, _) => x
+  def ι[A]: A => A                                           = x => x
```

Comment:
Can we keep the code base to ASCII only, please? I was only designed to handle 8-bit character sets.

Reply:
At the very least, be kind to editors and start the file with the encoding:
```scala
/**
 * -*- coding: utf-8; -*-
 *
 * This is the SKI combinator business that your mother warned you about.
 */
package coop.rchain.catscontrib

```

Reply:
@nashef @kirkwood really... once your eyes adjust, reading `something.map(κ("const"))` becomes neat & concise!

but I can rename it, if its a huge issue. will `k` &  `i` be ok? i will remove rho.

Reply:
Looking at SKI combinators in code gives me the same thrill as looking at a very large spider: both entranced and horrified at the same time.  

The k and i combinators are really easy to understand, though---it's only s that makes people have to stop and think for more than a second, so if you're getting rid of sigma (I think that's what you meant instead of rho), then I have no objection to using them.